google/src/main/scala/org/broadinstitute/dsde/rawls/metrics/GoogleInstrumented.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.metrics
+
+import com.google.api.client.googleapis.services.AbstractGoogleClientRequest
+import com.google.api.client.http.{HttpResponse, HttpResponseException}
+import nl.grons.metrics.scala.{Counter, Timer}
+import org.broadinstitute.dsde.rawls.metrics.GoogleInstrumented._
+import org.broadinstitute.dsde.rawls.metrics.GoogleInstrumentedService._
+
+/**
+  * Mixin trait for Google instrumentation.
+  */
+trait GoogleInstrumented extends WorkbenchInstrumented {
+  final val GoogleServiceMetricKey = "googleService"
+
+  protected implicit def googleCounters[A: GoogleInstrumentedServiceMapper]: GoogleCounters[A] =
+    (request, responseOrException, isRetry) => {
+      val base = ExpandedMetricBuilder
+        .expand(GoogleServiceMetricKey, implicitly[GoogleInstrumentedServiceMapper[A]].service)
```

Comment:
This is where we implicitly look up the `GoogleInstrumentedServiceMapper` typeclass.